### ISSUE TITLE
TypeError: client.unsafe is not a function

### ISSUE THREAD

**Issue by mvolkmann** [porsager/postgres#768]
This very simple demo of using Drizzle with PostgreSQL and Node gives me the error message in the title.
Am I doing something wrong in this file?
https://github.com/mvolkmann/drizzle-postgres-node/blob/main/src/db.mjs

**Comment by Louis-Tian** [comment]
The file you referenced does not have any reference to unsafe all, so I can't tell you what the exact problem is. But I would guess the problem is exactly as the "error message" is saying. "unsafe" is not a function. unsafe is a string template. You don't do `unsafe(...)` instead you do `` unsafe`...` ``.

**Comment by mvolkmann** [comment]
@Louis-Tian You are correct that my code does not attempt to use `unsafe`. That is happening somewhere in the library code.
